templates/404.html

```diff
@@ -0,0 +1,9 @@
+<!-- wp:template-part {"slug":"header","area":"header","tagName":"header"} /-->
+
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained","justifyContent":"left"}} -->
```

Comment:
```suggestion
<!-- wp:group {"tagName":"main","layout":{"type":"constrained"}} -->
```
@cliffralessio Could you please remove `"justifyContent":"left"` from the `404.html` file? 

The `"justifyContent":"left"` style is causing the 404 pattern to be left-aligned instead of centered.

Reply:
Please disregard this since this markup will be replaced.